lib/image-stream/supported.js

```diff
@@ -56,5 +56,9 @@ module.exports = [
   {
     extension: 'raw',
     type: 'image'
+  },
+  {
+    extension: 'dmg',
+    type: 'image'
```

Comment:
It can be raw, or compressed – that depends on how it was created. Wasn't sure what to go for... but I guess most of the time the contents would be compressed.